docs/developers/reference/api/linea-estimategas.mdx

```diff
@@ -146,3 +136,71 @@ decimals to get the wei value. You can use any hexadecimal to decimal converter
 [RapidTables](https://www.rapidtables.com/convert/number/hex-to-decimal.html).
 
 :::
+
+## Compatibility mode
+
+When `linea_estimateGas` is activated on Mainnet on September 30, infrastructure providers and those
+using their own nodes to submit transactions must adjust their configuration files to disable 
+compatibility mode.
+
+No action is required for Linea Sepolia nodes using the `advanced-testnet` profile from the 
+[`linea-besu-package` repository](https://github.com/Consensys/linea-besu-package/tree/main), since 
+`linea_estimateGas` is already activated on Linea Sepolia.
+
+### Effects 
+
+`linea_estimateGas` in compatibility mode returns the same gas price as `eth_gasPrice`. This means
+gas price is applied consistently regardless of the relative complexity of the transaction 
+(measured through its `calldata` size). As a result, transactions can be underpriced and risk 
+getting stuck, or be overpriced and result in the user overpaying in fees.
+
+This can be described with the following formula:
+```
+fee = gasPrice * gasUsed
+```
+
+### Benefits of disabling
+
+With compatibility mode disabled, `linea_estimateGas` is able to function as designed and return a 
+more accurate gas price, better suited to the transaction. The gas price scales with the amount of
+`calldata` a transaction contains. As a result, Linea can more effectively ensure the gas price
+reflects L1 costs and prover costs, which rise with transaction complexity/`calldata` size.
+
+The formula with `linea_estimateGas` enabled is:
+```
+fee = (calldataUsed * multiplier + constant) * gasUsed
```

Comment:
That's inaccurate
* multiplier - Linea's cost per byte on L1 in current market conditions
* There's another multiplier `(overheadMultiplier aka margin) * (calldataUsedInBytes * multiplier + constant) * gasUsed`. This margin is to ensure that in case of a sudden change of market conditions (gas price increase) transaction is still accepted

I know that public knowledge shouldn't be exactly the same as internal, but please take it into consideration